x-pack/plugin/ent-search/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/application/FullClusterRestartIT.java

```diff
@@ -57,7 +53,7 @@ public void testBehavioralAnalyticsDataRetention() throws Exception {
 
         assumeTrue(
             "Data retention changed by default to DSL in " + DSL_DEFAULT_RETENTION_VERSION,
-            getOldClusterTestVersion().before(DSL_DEFAULT_RETENTION_VERSION)
+            getOldClusterTestVersion().before(DSL_DEFAULT_RETENTION_VERSION.toString())
```

Comment:
Nit: if this does not work, you can use `getOldClusterVersion()`. But if it does work don't worry, I'm going to change it with https://github.com/elastic/elasticsearch/pull/104462 anyway